tests/ptvsd/server/test_attach.py

```diff
@@ -6,55 +6,47 @@
 
 import pytest
 
-from tests import code, debug, test_data
+from tests import debug, test_data
 from tests.patterns import some
 from tests.timeline import Event
 
 
-@pytest.mark.parametrize("wait_for_attach", ["waitOn", "waitOff"])
-@pytest.mark.parametrize("is_attached", ["attachCheckOn", "attachCheckOff"])
-@pytest.mark.parametrize("break_into", ["break", "pause"])
-def test_attach(run_as, wait_for_attach, is_attached, break_into):
+@pytest.mark.parametrize("wait_for_attach", ["wait_for_attach", ""])
+@pytest.mark.parametrize("is_attached", ["is_attached", ""])
+@pytest.mark.parametrize("break_into_debugger", ["break_into_debugger", ""])
```

Comment:
that should say "pause" (or something similar), since this test is doing `pause` if it is not `break_into_debugger`